src/journey-maps/angular/journey-maps.ts

```diff
@@ -796,6 +759,16 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
           );
           this._mapLayerFilterService.collectLvlLayers();
           this._levelSwitchService.switchLevel(this._levelSwitchService.selectedLevel);
+          if (this.journeyMapsRoutingOption?.journey) {
```

Comment:
I looked at how others solved this problem and found this solution here: https://github.com/mapbox/mapbox-gl-js/issues/4006#issuecomment-772462907

Maybe it would also be an option to copy certain sources from the old style to the new style? What do you think @mstankala?